docs/user-guide/configure.md

```diff
@@ -109,6 +109,31 @@ For example, the following rule configuration would substitute in custom message
 
 Alternately, you can write a [custom formatter](../developer-guide/formatters.md) for maximum control if you need serious customization.
 
+Experimental feature: some rules support message arguments. For example, when configuring the `color-no-hex` rule the hex color can be used in message string:
+
+`.stylelintrc.js`:
+
+```js
+{
+  'color-no-hex': [true, {
+    message: (hex) => `Don't use hex colors like "${hex}"`,
+  }]
+}
+```
+
+`.stylelintrc.json`:
+
+<!-- prettier-ignore -->
+```json
+{
+  "color-no-hex": [true, {
+    "message": "Don't use hex colors like \"%s\""
+  }]
+}
+```
+
+With formats that don't support a function like JSON, you can use a `printf`-like format (e.g., `%s`).
```

Comment:
[note] I think a description of `%s` is necessary. Please let me know if you have a better expression.

See also the [`util.format`](https://nodejs.org/api/util.html#utilformatformat-args) on the Node.js documentation.